prover/drivers/base.md

```diff
@@ -24,13 +24,14 @@ module PROVER-CONFIGURATION
         <k> $COMMANDLINE:CommandLine ~> $PGM:Pgm </k>
         <exit-code exit=""> 1 </exit-code>
         <goals>
-          <goal multiplicity="*" type="Set" format="%1%i%n%2, %3, %4%n%5%n%6%n%7%n%d%8">
+          <goal multiplicity="*" type="Set" format="%1%i%n%2, %3, %4%n%5%n%6%n%7%n%8%n%d%9">
             <active format="active: %2"> true:Bool </active>
             <id format="id: %2"> root </id>
             <parent format="parent: %2"> .K </parent>
             <claim> .K </claim>
             <strategy> .K </strategy>
             <expected> .K </expected>
+            <claim-name> .K </claim-name>
```

Comment:
Can we use `<id/>` for reference?

Reply:
We can't, because every user-provided claim has a `root` id.

Reply:
We can change this though. We can allow ids to hold both strings and integers.

Reply:
Currently, it would be simplest to allow only integers and strings, and use the auto-generated names like `ax12` for unnamed toplevel claims.